targets/pano_logic_g2/Makefile.mk

```diff
@@ -11,27 +11,84 @@ TARGET ?= $(DEFAULT_TARGET)
 COMM_PORT ?= /dev/ttyUSB0
 BAUD ?= 115200
 
+ifeq ($(FIRMWARE),linux)
+
+
+OVERRIDE_FIRMWARE=--override-firmware=none
+define fbi_rule
+$(1): $(subst .fbi,,$(1))
+endef
+
+TARGET_BUILD_DIR = build/$(FULL_PLATFORM)_$(TARGET)_$(FULL_CPU)
```

Comment:
this has already been defined in `Makefile:144`; is it needed again?

Reply:
Yes it is.  The Makefile.mk is included on line 120 of the top level make file so TARGET_BUILD_DIR isn't defined when it's included.  Additionally TARGET is defined my the Makefile.mk so TARGET_BUILD_DIR  can't be defined by the top level Makefile before the target makefile is included.

Reply:
That's fine then